ortools/constraint_solver/samples/cvrp_reload.py

```diff
@@ -121,12 +121,15 @@ def manhattan_distance(position_1, position_2):
 def create_distance_evaluator(data):
     """Creates callback to return distance between points."""
     _distances = {}
+    BIG_DISTANCE=10000
```

Comment:
let's move it in data so you can reuse it as distance dimension horizon
note: this is what I have done on master